cfgov/jinja2/v1/about-us/the-bureau/bureau-structure/index.html

```diff
@@ -46,9 +46,9 @@
 
 {% block content_main %}
     <h1>Bureau Structure</h1>
-    <p>Org chart last updated: August 29, 2016</p>
-    <div class="org-chart">
-        <div class="org-chart_node org-chart_node__root">
+    <p>Org chart last updated: November 8, 2016</p>
```

Comment:
Yeah, this file will be deleted once the bureau structure is migrated to Wagtail.